packages/cli/src/hbs_extractor.ts

```diff
@@ -4,8 +4,8 @@ function extractText(node: any, fileName: any, options: any) {
   if (node.path.original === 'format-message') {
     let message = node.params[0]?.original
     let desc = node.params[1]?.original
-    let id = options.overrideIdFn(undefined, message, desc, fileName)
     let defaultMessage = message?.trim().replace(/\s+/gm, ' ')
+    let id = options.overrideIdFn(undefined, defaultMessage, desc, fileName)
```

Comment:
Maybe add a code comment here to make clear that these have to happen in this specific order